addons/web_editor/models/ir_ui_view.py

```diff
@@ -132,8 +132,10 @@ def _copy_custom_snippet_translations(self, record, html_field):
 
     @api.model
     def _copy_field_terms_translations(self, record_from, name_field_from, record_to, name_field_to):
-        """ Copy the terms translation from a record/field ``Model1.Field1``
-        to a (possibly) completely different record/field ``Model2.Field2``.
+        """ Copy the `model_terms` translations from `record_from.name_field_from`
```

Comment:
NIT don't quote the `model_terms`, it is a terminology of ORM which means the `field.translate` is callable. Maybe direct use: model terms translations